xmake/modules/package/tools/cmake.lua

```diff
@@ -1110,7 +1110,8 @@ function _build_for_ninja(package, configs, opt)
     _fix_pdbdir_for_ninja(package)
     ninja.build(package, {}, {envs = opt.envs or buildenvs(package, opt),
         jobs = opt.jobs,
-        target = opt.target})
+        target = opt.target,
+        targets = opt.targets})
```

Comment:
why, opt.target support multiple targets.

If you think the name is inappropriate, you can use both targets and target instead of passing two parameters at the same time.

```
targets = opt.targets or opt.target
```